app/views/spree/admin/reports/_table.html.haml

```diff
@@ -20,4 +20,4 @@
           %td{colspan: @header.count}= t(:none)
 - else
   %p.report__message
-    = t(".select_and_search")
+    = t(".select_and_search", option: msg_option.upcase)
```

Comment:
The passed `msg_option` needs to be translated, otherwise it will stay in English while the rest of the sentence changes.